service/frontend/configs/quotas.go

```diff
@@ -205,7 +205,7 @@ func NewNamespaceRateBurst(
 		namespaceName: namespaceName,
 		rateFn:        rateFn,
 		burstFn: func(namespace string) int {
-			return int(rateFn(namespace) * math.Max(1, burstRatioFn(namespace)))
+			return int(math.Max(1, rateFn(namespace)*burstRatioFn(namespace)))
```

Comment:
I would write it like
```suggestion
			return max(1, math.Ceil(rateFn(namespace)*burstRatioFn(namespace)))
```
to do the max as integers, and also round up